src/App.js

```diff
@@ -1,18 +1,14 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
 import './App.css';
+import Title from './components/Title';
+import Header from './components/Header';
 
 class App extends Component {
```

Comment:
This can be a functional component since it doesn't have state or use any lifecycle methods.
```jsx
export default function App () {
  return (
    <div className="App">
       ...
    </div>
  );
}
 ```